tests/test_point_range.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for Point.
+
+To run this unit test install UnitTesting package and choose
+"UnitTesting: Test current project" command.
+"""
+
+from unittest import TestCase
+
+from LSP.main import Point
+from LSP.main import Range
```

Comment:
Was mypy working in this file? It should prevent you from sending `None` in for dict, for example?

Reply:
How is your question related with line 11?